modules/nf-core/wipertools/fastqgather/meta.yml

```diff
@@ -0,0 +1,58 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "wipertools_fastqgather"
+description: A tool of the wipertools suite that merges FASTQ chunks produced by wipertools_fastqscatter
+keywords:
+  - fastq
+  - merge
+  - union
+tools:
+  - "fastqgather":
+      description:
+        "A tool of the wipertools suite that merges FASTQ chunks produced\
+        \ by wipertools_fastqscatter."
+      homepage: "https://github.com/mazzalab/fastqwiper"
+      documentation: "https://github.com/mazzalab/fastqwiper"
+      tool_dev_url: "https://github.com/mazzalab/fastqwiper"
+      doi: "no DOI available"
+      licence: ["GPL v2-or-later"]
+      identifier: ""
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test', single_end:false ]
```

Comment:
Is this applicable (so do you use information on single_end vs. paired_end fastqs)?